Sources/BraintreeCore/BTPaymentMethodNonceParser.swift

```diff
@@ -53,18 +53,80 @@ import Foundation
             return nil
         }
 
-        if let completionHandler = completionHandler {
+        if let completionHandler {
             return completionHandler(json)
         }
 
-        if json?["nonce"].isString != false {
+        if json?["nonce"].isString == false {
+            return nil
+        }
+
+        let type = json?["type"].asString()
+
+        if type == "CreditCard", let cardType = json?["details"]["cardType"].asString() {
+            return BTPaymentMethodNonce(
+                nonce: json?["nonce"].asString() ?? "",
+                type: self.cardType(from: cardType),
+                isDefault: json?["default"].isTrue ?? false
+            )
+        } else if type == "ApplePayCard" {
+            return BTPaymentMethodNonce(
+                nonce: json?["nonce"].asString() ?? "",
+                type: json?["details"]["cardType"].asString() ?? "ApplePayCard",
+                isDefault: json?["default"].isTrue ?? false
+            )
+        } else if type == "PayPalAccount" {
+            return BTPaymentMethodNonce(
+                nonce: json?["nonce"].asString() ?? "",
+                type: "PayPal",
```

Comment:
Same comment here, re using `PayPalAccount` in [V5's load method](https://github.com/braintree/braintree_ios/blob/75a48494b48dc83204ced72a0cee68057a8e7170/Sources/BraintreePayPal/BTPayPalDriver.m#L79-L81)

```suggestion
                type: "PayPalAccount",
```

Reply:
Same as Venmo, under the hood we pass the type as PayPal per [this test](https://github.com/braintree/braintree_ios/blob/75a48494b48dc83204ced72a0cee68057a8e7170/UnitTests/BraintreePayPalTests/BTPaymentMethodNonceParser_PayPal_Tests.swift#L22) and not what's passed into the register method. Verified in the demo app in v5 as well.